src/Simplex_tree/include/gudhi/Simplex_tree.h

```diff
@@ -1017,9 +1065,16 @@ class Simplex_tree {
    * \param simplex represent the simplex of which we search the star
    * \return Vector of Simplex_handle, empty vector if no cofaces found.
```

Comment:
Not necessarily a vector.

Reply:
It was no more the case, but thanks to [5104751](https://github.com/GUDHI/gudhi-devel/pull/877/commits/5104751ff686175d30b0089ddb20f6f21a108fa9) I rolled back the return type of star_simplex_range.

Reply:
So now again it isn't necessarily a vector?

Reply:
I specified cofaces/stars_simplex_range return type in function of link_nodes_by_label on [a27f109](https://github.com/GUDHI/gudhi-devel/pull/877/commits/a27f1097a207f4d3eb7a496227f2d0773195be6b)

Reply:
We don't have to be that specific (we don't want users to rely on the fact that it is a vector in some case), but that will do for now.